src/verify.v2.integration.test.ts

```diff
@@ -612,18 +612,18 @@ describe("verify(integration)", () => {
     expect(isValid(results)).toStrictEqual(true);
   });
   it("should be invalid with a merkle root that is odd-length", async () => {
+    // TODO some fragments are skipped because merkleRoot is weird. Do we want that to happen ? result is slightly different
```

Comment:
i think skipped is weird if the object key is correctly formed but the merkle root is corrupt

Reply:
fixed